portfolio/src/main/java/com/google/sps/servlets/ContactMeFormSubmissionServlet.java

```diff
@@ -0,0 +1,16 @@
+package com.google.sps.servlets;
+
+import java.io.*;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.*;
+
+
+@WebServlet("/submitForm")
+public class ContactMeFormSubmissionServlet extends HttpServlet{
```

Comment:
You need a space "Before any open curly brace"

https://google.github.io/styleguide/javaguide.html#s4.6.2-horizontal-whitespace